src/pages/filter/ui/panels/function-panel/components/inheritance-mode/complex-variants.tsx

```diff
@@ -20,44 +19,39 @@ interface IComplexVariantsProps {
 
 export const ComplexVariants = observer(
   ({
-    variantsValues,
-    problemGroupValues,
-    filteredComplexVariants,
+    variantValues,
+    variants,
     handleChangeVariants,
   }: IComplexVariantsProps) => {
     return (
       <React.Fragment>
         <div className="flex items-center justify-between">
           <p className="text-14 leading-14px text-grey-blue">
-            {variantsValues.length} Selected
+            {variantValues.length} Selected
           </p>
 
           <span
             className="text-12 leading-14px text-blue-bright cursor-pointer ml-auto mr-2"
-            onClick={() =>
-              inheritanceModeStore.handleSelectAllVariants(problemGroupValues)
-            }
+            onClick={() => inheritanceModeStore.selectAllVariants()}
```

Comment:
{inheritanceModeStore.selectAllVariants}

Reply:
Does not work